gobblin-metrics/src/test/java/gobblin/metrics/influxdb/InfluxDBReporterTest.java

```diff
@@ -177,7 +167,7 @@ public void tearDown() {
    */
   private static class TestInfluxDB extends InfluxDBImpl {
```

Comment:
I couldn't find anything through a quick google search, but do you know if there are already implemented InfluxDB mocks?